ledger/participant-state/kvutils/src/test/suite/scala/com/daml/ledger/participant/state/kvutils/committer/transaction/TransactionCommitterSpec.scala

```diff
@@ -595,58 +595,125 @@ class TransactionCommitterSpec extends AnyWordSpec with Matchers with MockitoSug
   }
 
   "validateContractKeys" should {
-    "return Inconsistent when a contract key resolves to a different contract ID than submitted by a participant" in {
-      def createNode(contractId: String): Create =
-        create(contractId, signatories = Seq("Alice"), keyAndMaintainer = Some(aKey -> "Alice"))
+    def createNode(contractId: String): Create =
```

Comment:
Since you're already here :) Perhaps this helps clarifying a bit:

```suggestion
    def newCreateNodeWithFixedKey(contractId: String): Create =
```